lib/presentation/mixins/send_files_mixin.dart

```diff
@@ -68,6 +70,24 @@ mixin SendFilesMixin {
     return result.files.map((file) => file.toMatrixFileOnWeb()).toList();
   }
 
+  Future<List<MatrixFile>> pickFilesFromDesktop() async {
+    final String initialDirectory =
+        (await getApplicationDocumentsDirectory()).path;
+    final List<XFile> xFiles =
+        await openFiles(initialDirectory: initialDirectory);
```

Comment:
IMO: I saw you still using it in another place (lib/pages/settings_dashboard/settings_profile/settings_profile.dart). Can you separate this func so it can be reused?

Reply:
IMO: If `(await getApplicationDocumentsDirectory()).path` or `await openFiles(initialDirectory: initialDirectory)`
has error, how to handle error?

Reply:
> IMO: I saw you still using it in another place (lib/pages/settings_dashboard/settings_profile/settings_profile.dart). Can you separate this func so it can be reused?

Actually on `lib/pages/settings_dashboard/settings_profile/settings_profile.dart` the method used is `openFile` without the `s` at the end of the word file. This method force user to open only one file.

But `openFiles` used here allow user to select multiple files

Reply:
> IMO:
> 
> * If `(await getApplicationDocumentsDirectory()).path` or `await openFiles(initialDirectory: initialDirectory)`
>   has error, how to handle error?
> * Should try catch for this case, WDYT?

done